CMakeLists.txt

```diff
@@ -26,18 +26,25 @@ option(OQS_BUILD_ONLY_LIB "Build only liboqs and do not expose build targets for
 set(OQS_MINIMAL_BUILD "" CACHE STRING "Only build specifically listed algorithms.")
 option(OQS_PERMIT_UNSUPPORTED_ARCHITECTURE "Permit compilation on an an unsupported architecture." OFF)
 option(OQS_STRICT_WARNINGS "Enable all compiler warnings." OFF)
+option(OQS_POSITION_INDEPENDENT_CODE "Generate Position Independent Code (-fPIC)" ON)
+option(OQS_USE_EXTERNAL_RNG "Use an external RNG instead of the default system RNG." OFF)
 
 set(OQS_OPT_TARGET auto CACHE STRING "The target microarchitecture for optimization.")
 
 set(CMAKE_C_STANDARD 11)
 set(CMAKE_C_STANDARD_REQUIRED ON)
-set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 set(CMAKE_C_VISIBILITY_PRESET hidden)
 set(OQS_VERSION_TEXT "0.10.0-dev")
 set(OQS_COMPILE_BUILD_TARGET "${CMAKE_SYSTEM_PROCESSOR}-${CMAKE_HOST_SYSTEM}")
 set(OQS_MINIMAL_GCC_VERSION "7.1.0")
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+if(OQS_POSITION_INDEPENDENT_CODE)
```

Comment:
Why is this introduced -- to then disable it in the Zephyr section? Couldn't this be avoided by simply setting "CMAKE_POSITION_INDEPENDENT_CODE" to "OFF" in the Zephyr `cmake` files?

Reply:
I will look into that tomorrow and update the commit.

Reply:
I just removed the change regarding the `POSITION_INDEPENDENT_CODE` option. It is now disabled from within the CMakeLists.txt in the zephyr directory.